src/utils.ts

```diff
@@ -69,7 +70,9 @@ export async function postSlackNotification (
     }
 
     // message formatting reference - https://api.slack.com/reference/surfaces/formatting
-    const text = `<${repoUrl}|${repo.repo}> deployment 🚀 to <${deploymentUrl}|${environment}> by <@${actor.toLowerCase()}> completed with ${status} ${statusIcon} - ${commitText}`
+    const baseText = `<${repoUrl}|${repo.repo}> deployment 🚀 to <${deploymentUrl}|${environment}> by <@${actor.toLowerCase()}> completed with ${status} ${statusIcon} - ${commitText}.`
+    const text = status === 'success' ? `${baseText}\n\n- :toolbox: Check out our <${dataDogDeploymentConfidenceDashboard}|**deployment confidence dashboard**> so you are the first to know if anything is broken.` : baseText
```

Comment:
so if the intention is to provide the link to confidence dashboard on every deployment, i would just add that link to the slack channel description. Imagine you run dozens of deployments per day so the #deploy-ops channel can really become spammy with the same message.

We tried to make those slack messages minimalistic exactly to keep the channel scrollable. Does this make sense?

Reply:
That is a good shout, hadn't really thought of that alternative! @hfaulds WDYT?

Reply:
I understand the rationale but personally I prefer linking repeatedly as part of the `@` mention to the user. This is what heaven does when deploying moda apps and really reinforces that this is what you should be looking at during a deploy, whereas I find slack channel descriptions very ignorable

Reply:
Tend to agree, plus if we find that it is too spammy, we can always revert it back 😊

Reply:
Sure, whatever makes more sense for the devs who will use those.